backend/libexecution/libjson.ml

```diff
@@ -35,7 +35,7 @@ let fns : fn list =
               fail args)
     ; preview_safety = Safe
     ; deprecated = true }
-  ; { prefix_names = ["JSON::parse_v0"]
+  ; { prefix_names = ["JSON::parse"]
```

Comment:
I think it would be better to not do this rename, as it might change something in the code. Can this be avoided?

Reply:
I think it's necessary because the test cases for this Ocaml function will cause problems. Something that I realized working with it and after some debug if I leave `JSON::parse_v0` the test for Ocaml doesn't recognize the name of the function. Also, I took a look at other functions of the Ocaml version and the first version of a function (or version 0) doesn't have `_v0` just the name function. I think maybe it was a typo?

Reply:
Yeah, that's nasty. I had to deal with this before and I think the same solution will work for you.

I think you should be able to leave the code in backend as it was (with the `_v0` suffix), and just add an if statement like this one:

https://github.com/darklang/dark/blob/main/fsharp-backend/src/ApiServer/Functions.fs#L87